src/interpreter.cpp

```diff
@@ -864,6 +865,7 @@ struct command_info cmd_info[] =
     { "spool"    , POS_DEAD    , do_spool    , 0, 0, FREE }, 
     { "speed"    , POS_RESTING , do_speed    , 0, 0 },
     { "spells"   , POS_SLEEPING, do_spells   , 0, 0 },
+    { "spellset" , POS_SLEEPING, do_spellset , LVL_DEVELOPER, 0 },
```

Comment:
Should this require higher privs? I don't recall what 'developer' is numerically.